packages/kas/README.md

```diff
@@ -1,11 +1,9 @@
-KAS
-===
+# @khanacademy/kas
```

Comment:
Nice. I like using the actual package name instead of yet another "name" for the readme.